index.js

```diff
@@ -155,14 +155,24 @@ Notifications.localNotification = function(details: Object) {
  */
 Notifications.localNotificationSchedule = function(details: Object) {
 	if ( Platform.OS === 'ios' ) {
-		this.handler.scheduleLocalNotification({
+		const iosDetails = {
 			fireDate: details.date.toISOString(),
 			alertBody: details.message,
-			userInfo: details.userInfo
-		});
+			userInfo: details.userInfo,
+			repeatInterval: details.repeatType
```

Comment:
Hello @eujern , we've made almost the same pull request [mine here](https://github.com/zo0r/react-native-push-notification/pull/288). But there is a couple of things (the deletes), in my opinion that may not be necessary. I'd like to know why you added them...